documentation/book/assembly-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-maintenance-time-windows-{context}']
+
+= Maintenance time windows
+
+The are different scenarios for which the Kafka and the Zookeeper clusters need a rolling update.
```

Comment:
Maintenance time windows is a feature that for now supports only cluster CA certificate renewal. It means that because the renewal needs a rolling update you can define "when" it can happen (because you don't want the nodes shut down in some "hot" times). We thought that it could be useful for other things that can raise a rolling update but for now only CA renewal is supported.

Reply:
@ncbaratta I am fine to rewrite the paragraph as you suggested but we need to specify that it works just for rolling updates triggered by a cluster CA certificate renewal. Otherwise people think that it's useful in other cases.

Reply:
> now supports only cluster CA certificate renewal

We support it for clients CA renewal too, right?

Reply:
That makes perfect sense!  So, I'd add an : 

IMPORTANT: Currently, the only supported maintenance time window is related to CA certificate renewals.

Reply:
@tombentley absolutely true!

Reply:
@ncbaratta because the maintenance time windows feature suppors a list of windows, can we say:

IMPORTANT: Currently, the maintenance time windows are supported only for CA certificate renewals

or something like that? from your statement it seems that just one window can be specified.

Reply:
Yes! Small edits : 

IMPORTANT: Currently, maintenance time windows are only supported for CA certificate renewals.